CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+##2016-10-03 - Supported Release 2.0.3
+###Summary
+
+Small release with bugs fixes and another speed improvement.
+
+###Bug Fixes
+- Miscellanous fixes which improve reliability
+- Address json pure version problems ([MODULES-3640](https://tickets.puppetlabs.com/browse/MODULES-3640))
+- Capture exit codes when executing external scripts ([MODULES-3399](https://tickets.puppetlabs.com/browse/MODULES-3399))
+- Update appveyor and travis to use Ruby 2.3 ([MODULES-3775](https://tickets.puppetlabs.com/browse/MODULES-3775))
```

Comment:
Actually remove this one, not important to the changelog (present functionality of the module itself)